Tests/Twig/Extension/SonataAdminExtensionTest.php

```diff
@@ -340,6 +342,16 @@ public function testDeprecatedRenderListElement($expected, $value, array $option
         );
     }
 
+    public function buildUrl($url)
+    {
+        if (defined('PHP_QUERY_RFC3986')) {
+            // add htmlspecialchars because twig add it auto
```

Comment:
Since php5.4 exist const 'PHP_QUERY_RFC3986' and twig using it in url_encode.

```
/**
 * URL encodes (RFC 3986) a string as a path segment or an array as a query string.
 *
 * @param string|array $url A URL or an array of query parameters
 *
 * @return string The URL encoded value
 */
function twig_urlencode_filter($url)
{
    if (is_array($url)) {
        if (defined('PHP_QUERY_RFC3986')) {
            return http_build_query($url, '', '&', PHP_QUERY_RFC3986);
        }

        return http_build_query($url, '', '&');
    }

    return rawurlencode($url);
}
```

What's why my test case fail on php 5.3

@greg0ire please look https://3v4l.org/sX7Dn